src/main/java/com/google/api/generator/engine/ast/AnnotationNode.java

```diff
@@ -62,12 +66,16 @@ public abstract static class Builder {
     public abstract Builder setType(TypeNode type);
 
     public Builder setDescription(String description) {
-      return setDescriptionExpr(ValueExpr.withValue(StringObjectValue.withValue(description)));
+      return setDescriptionExprs(
+          Arrays.asList(ValueExpr.withValue(StringObjectValue.withValue(description))));
+    }
+
+    public Builder setDescriptions(List<Expr> exprList) {
```

Comment:
I like this improvement in general, but we should be careful exposing a setter that can set descriptions to any `Expr`, you can technically pass a `NewObjectExpr` or a `ReturnExpr` and everything will work until we try to compile the generated code, which could be too late to find an issue. 
Correct me if I'm wrong, I think we only allow `ValueExpr`, `AssignmentExpr`, or `VariableExpr`(for clazz if you remove `CLazzValue`). So we could either add some type checks in the `build()` method, or only expose methods to add descriptions for the above types, like `addDescription(ValueExpr expr)`, `addDescription(AssignmentExpr expr)` and `addDescription(VariableExpr expr)`.
Basically anything that could restrict people from passing any types of `Expr` would be great.

Reply:
Good point, let me refactor this part and expose methods for each expr type.